builtin/logical/totp/path_code.go

```diff
@@ -85,8 +86,8 @@ func (b *backend) pathReadCode(ctx context.Context, req *logical.Request, data *
 }
 
 func (b *backend) pathValidateCode(ctx context.Context, req *logical.Request, data *framework.FieldData) (*logical.Response, error) {
-	name := data.Get("name").(string)
-	code := data.Get("code").(string)
+	name := strings.TrimSpace(data.Get("name").(string))
```

Comment:
are we sure this is ok? like we use `name` as the key, so could it be considered a breaking change for the same request to now be looking for a different key?